Applications/Viqui/vqApplication.cxx

```diff
@@ -1112,6 +1116,21 @@ void vqApplication::generateReport()
     }
 }
 
+//-----------------------------------------------------------------------------
+void vqApplication::exportSvm()
```

Comment:
I think it would be more consistent to have this in `vqCore` (ideally between `saveQueryPlan` and `saveResults`). This also eliminates the need to add other utility methods to `vqCore`. (Or to touch `vqApplication.h` at all.) Although keeping `updateIqrModel`, but as a *protected* method, probably makes sense.